### PR TITLE
fix graphql field creation, ignore mandatory id field

### LINKED ISSUE #10451: Graphql mutation "create_fields_item" can't create any field because we must provide an Id for it
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

Hello,
i would like to create fields with graphql. However, i must provide an id.

Same issue when creating fields with a collection **create_collections_item**
https://docs.directus.io/reference/system/collections/#create-a-collection

Relatives issues #5859  #5723  #5931 #5875
thx

### To Reproduce

```graphql
mutation {
  create_fields_item(
    data: {
      collection: "subscription"
      field: "date"
      type: "string"
      meta: {
        readonly: false
        hidden: false
        translations: [
          { language: "en-US", translation: "Date" }
        ]
        special: []
        interface: "input"
        collection: "subscription"
        field: "date"
      }
      schema: { name: "date" }
    }
    collection: "subscription"
  ) {
    field
  }
}

```

### Errors Shown

```json
{
    "errors": [
        {
            "message": "GraphQL validation error.",
            "extensions": {
                "code": "GRAPHQL_VALIDATION_EXCEPTION",
                "graphqlErrors": [
                    {
                        "message": "Field \"directus_fields_meta_input.id\" of required type \"Int!\" was not provided.",
                        "locations": [
                            {
                                "line": 7,
                                "column": 13
                            }
                        ]
                    }
                ]
            }
        }
    ]
}
```

### What version of Directus are you using?

9.2.2

### What version of Node.js are you using?

v15.14.0

### What database are you using?

sqlite

### What browser are you using?

chrome,brave

### What operating system are you using?

linux

### How are you deploying Directus?

running locally

### PR DESCRIPTION
**Pull request by sohelzerdoumi**
Fixes #10451

### CODE REVIEW

File: api/src/services/graphql.ts
Comment (rijkvanzanten) [inline api/src/services/graphql.ts:2108]: We'll have to check for UUID generation as well as auto increment

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: api/src/services/graphql.ts
Comment (sohelzerdoumi) [reply api/src/services/graphql.ts]: done on  commit 4dd93b9

**Comment by sohelzerdoumi** [comment]
i need this fix in order to auto-build directus applications

**Comment by jaycammarano** [comment]
> i need this fix in order to auto-build directus applications

Highly recommend merging main into your branch. Highly recommend opening this up to maintainers so I can write tests OR (even better) writing tests yourself.

Would love for this to be merged before #11213 if we merge it.

**Comment by rijkvanzanten** [comment]
Closing this as commits are generated under a non-github user handle. Lets treat the fix as part of https://github.com/directus/directus/issues/10610

**Pull request closed without merging**
